modules/serverless-scale-to-zero-grace-period.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-admin-autoscaling.adoc
+
+[id="serverless-scale-to-zero-grace-period_{context}"]
+= Configuring the scale-to-zero grace period
+
+This setting specifies an upper bound time limit that the system waits for scale-from-zero machinery to be in place before the last replica of an application is removed.
+
+.Prerequisites
+
+* {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+* You have cluster administrator permissions.
+* You are using the default Knative Pod Autoscaler. The scale to zero feature is not available if you are using the Kubernetes Horizontal Pod Autoscaler.
+
+.Procedure
+
+* Modify the `scale-to-zero-grace-period` spec in the `KnativeServing` CR:
```

Comment:
Are we going to add other config options in the future like https://knative.dev/docs/serving/autoscaling/scale-to-zero/#scale-to-zero-last-pod-retention-period? Same applies for all other options like rps-target etc.
Eventually are we copying the upstream stuff here or do we just provide some basic config options to get people started?

Reply:
Yes I will add them all eventually (and improve upstream as we have time), but it won't be all at once because we have limited resources on the docs team. For now I'm just trying to provide the basics a user might need to get started :) If ya'll have suggestions for other user stories I would love to hear what you think is important for users?